Library/Formula/snappystream.rb

```diff
@@ -14,9 +14,10 @@ class Snappystream < Formula
 
   depends_on "cmake" => :build
   depends_on "snappy"
+  depends_on "boost"
 
   def install
-    system "cmake", ".", "-DBUILD_TESTS=ON", *std_cmake_args
+    system "cmake", ".", "-DBUILD_TESTS=ON", "-DWITH_BOOST_IOSTREAMS=1", *std_cmake_args
```

Comment:
No, it just enables linking with boost and adds preprocessor defines to the project.